components/datadog/updater/install_script.sh

```diff
@@ -10,14 +10,7 @@ if [ "$UID" == "0" ]; then
 else
     sudo_cmd="sudo"
 fi
-config_file="/etc/datadog-agent/datadog.yaml"
```

Comment:
you need to keep the config file to avoid breaking tests with Agent unit assertions in datadog-agent installer E2E suite

Reply:
removing it will need a first migration step in datadog-agent